test/e2e/specs/wp-signup-spec.js

```diff
@@ -78,117 +78,118 @@ before( async function () {
-			return await pickAPlanPage.selectFreePlan();
-		} );
-
-		step(
-			'Can then see the sign up processing page which will finish automatically move along',
-			async function () {
-				return await new SignUpStep( driver ).continueAlong( blogName, passwordForTestAccounts );
-			}
-		);
-
-		sharedSteps.canSeeTheOnboardingChecklist();
-
-		step( 'Can log out and request a magic link', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-			await driverManager.ensureNotLoggedIn( driver );
-			const loginPage = await LoginPage.Visit( driver );
-			return await loginPage.requestMagicLink( emailAddress );
-		} );
-
-		step( 'Can see email containing magic link', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-			const emailClient = new EmailClient( signupInboxId );
-			const validator = ( emails ) =>
-				emails.find( ( email ) => email.subject.includes( 'WordPress.com' ) );
-			const emails = await emailClient.pollEmailsByRecipient( emailAddress, validator );
-			assert.strictEqual(
-				emails.length,
-				2,
-				'The number of newly registered emails is not equal to 2 (activation and magic link)'
-			);
-			for ( const email of emails ) {
-				if ( email.subject.includes( 'WordPress.com' ) ) {
-					return ( magicLoginLink = email.html.links[ 0 ].href );
-				}
-			}
-			return assert(
-				magicLoginLink !== undefined,
-				'Could not locate the magic login link email link'
-			);
-		} );
-
-		step( 'Can visit the magic link and we should be logged in', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-			await driver.get( magicLoginLink );
-			const magicLoginPage = await MagicLoginPage.Expect( driver );
-			await magicLoginPage.finishLogin();
-			try {
-				await CustomerHomePage.Expect( driver );
-			} catch ( e ) {
-				await ReaderPage.Expect( driver );
-			}
-		} );
-
-		after( 'Can delete our newly created account', async function () {
-			return await new DeleteAccountFlow( driver ).deleteAccount( blogName );
-		} );
-	} );
+	// eslint-disable-next-line jest/no-commented-out-tests
+	// describe( 'Sign up for a free WordPress.com site from the Jetpack new site page, and log in via a magic link @signup @email', function () {
+	// 	const blogName = dataHelper.getNewBlogName();
+	// 	// const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
+	// 	const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
+	// 	let magicLoginLink;
+
+	// 	before( async function () {
+	// 		return await driverManager.ensureNotLoggedIn( driver );
+	// 	} );
+
+	// 	step(
+	// 		'Can visit the Jetpack Add New Site page and choose "Create a shiny new WordPress.com site"',
+	// 		async function () {
+	// 			const jetpackAddNewSitePage = await JetpackAddNewSitePage.Visit( driver );
+	// 			await jetpackAddNewSitePage.createNewWordPressDotComSite();
+	// 		}
+	// 	);
```

Comment:
This test would need modifications and be enabled. We could start directly from `/start`, I think it's something like this:

https://github.com/Automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/test/e2e/lib/flows/sign-up-flow.js#L35

Just without flow param.